src/Translate.js

```diff
@@ -50,28 +50,20 @@ class WrappedTranslate extends React.Component<TranslateWithContextProps> {
     const fallbackRenderToStaticMarkup = value => value;
     const renderToStaticMarkup =
       context.renderToStaticMarkup || fallbackRenderToStaticMarkup;
+    const hasId = id !== undefined;
+    const hasDefaultLanguage = defaultLanguage !== undefined;
+    const hasChildren = children === undefined;
```

Comment:
This seems confusingly named, since it is checking that children is `undefined`. It also means the if condition below reads as `if (hasChildren...) return` which isn't the case?

Reply:
Yep that’s my bad 🤗. Will look at renaming some of these.